hack/ci/e2e-conformance/e2e-conformance_patch.yaml.tpl

```diff
@@ -13,6 +13,59 @@ spec:
         kubeletExtraArgs:
           v: "8"
     verbosity: 8
+    preKubeadmCommands:
```

Comment:
@hidekazuna I asked at https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/613/files
about why we need remove this and you mentioned those code removal in on purpose
so actually can I think we instead remove all contents here, actually we should replace some code (e.g update kubectl version etc) ?

Reply:
I thought removing codes about CI VERSION is good at that time. Actually the code has two parts: binaries and images. binaries are must to update in preKubeadmCommands, I understand.

Reply:
ok ,let's wait for CI :)